deploy/GetIntoLiquidatablePosition.js

```diff
@@ -6,7 +6,6 @@ module.exports = async function (hre) {
     const { deployer, acc1 } = await getNamedAccounts()
     const signers = await ethers.getSigners()
 
-    await deployments.fixture("FullDeployTest")
     let tracerInstance = new ethers.Contract(
         await deployments.read("TracerPerpetualsFactory", "tracersByIndex", 0),
         perpsAbi
```

Comment:
I'm wondering how this works without getting a deployment fixture. I thought `deployments.read` requires you to have a fixture.